platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectLearner.java

```diff
@@ -182,8 +183,10 @@ private ReservedSignedState reconnect() throws InterruptedException {
         connection.getDis().getSyncByteCounter().resetCount();
         connection.getDos().getSyncByteCounter().resetCount();
 
+        final ReconnectConfig reconnectConfig =
+                platformContext.getConfiguration().getConfigData(ReconnectConfig.class);
         final LearningSynchronizer synchronizer =
-                new LearningSynchronizer(threadManager, in, out, currentState, connection::disconnect);
+                new LearningSynchronizer(threadManager, in, out, currentState, connection::disconnect, reconnectConfig);
```

Comment:
If we used the paltform context, we could just store this class's version and we wouldn't have to worry about extracting data for the constructor.